### PR TITLE
Adding a hometown file for belfast

### LINKED ISSUE #5: scottmcelveen Hometown
## Belfast, Northern Ireland
- [x] Create branch
- [x] Add my file
- [x] Commit my file
- [x] Open a pull request
- [ ] Get :+1: 
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by scottmcelveen**
This Pull request closes #5 and adds information about scottmcelveen's hometown!

### CODE REVIEW

File: belfast.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Belfast, Northern Ireland
+
+ - Filming location for Game of Thrones
```
Comment (scottmcelveen) [inline belfast.md:3]: What is game of thrones? @aruddell

Comment (aruddell) [inline belfast.md:3]: Never watched it @scottmcelveen

File: belfast.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+ - Go to eat at Burger Shack
```
Comment (aruddell) [inline belfast.md:4]: Is that the same as the Love Shack or is that different?
Do you like that song though @scottmcelveen ?
What do you give it out of :100:

**Comment by aruddell** [comment]
You didnt answer my question.
But you outrank me so :+1:

**Pull request merged**